include/swift/Basic/Statistic.h

```diff
@@ -0,0 +1,22 @@
+//===--- Statistic.h - Helpers for Lllvm::Statistic ---------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_BASIC_STATISTIC_H
+#define SWIFT_BASIC_STATISTIC_H
+
+#include "llvm/ADT/Statistic.h"
+
+#define SWIFT_FUNC_STAT \
```

Comment:
Does this need a `do { ... } while (0)` given how often braceless control flow is used, or are macro hacks like that not Swift/LLVM style?